multi-educator/src/Dscp/Educator/CLI.hs

```diff
@@ -0,0 +1,99 @@
+{-# LANGUAGE ApplicativeDo    #-}
+{-# LANGUAGE OverloadedLabels #-}
```

Comment:
My gosh seriously, why? It's obvious this can just be imported from `disciplina-educator`, wtf